inc/analyzers.json

```diff
@@ -116,7 +116,7 @@
 		},
 		"en_stem_filter": {
 			"type": "stemmer",
-			"name": "minimal_english"
+			"name": "light_english"
```

Comment:
Is this going to require reindexing?

Reply:
It'll continue to function without reindexing but the results may change a little bit after recreating the indexes. I found this was better.